patches/chrome-installer-linux-debian-postinst.patch

```diff
@@ -1,18 +0,0 @@
-diff --git a/chrome/installer/linux/debian/postinst b/chrome/installer/linux/debian/postinst
-index b284f042a0bdcc9964037ad636bd5e5fa49e7daa..8707a5b62c30508645dcea129d39597c2deef57c 100755
---- a/chrome/installer/linux/debian/postinst
-+++ b/chrome/installer/linux/debian/postinst
-@@ -49,8 +49,11 @@ update-alternatives --install /usr/bin/x-www-browser x-www-browser \
- update-alternatives --install /usr/bin/gnome-www-browser gnome-www-browser \
-   /usr/bin/@@USR_BIN_SYMLINK_NAME@@ $PRIORITY
- 
--update-alternatives --install /usr/bin/@@PACKAGE_ORIG@@ @@PACKAGE_ORIG@@ \
--  /usr/bin/@@USR_BIN_SYMLINK_NAME@@ $PRIORITY
-+if [ '"@@PACKAGE_ORIG@@"' != '"@@USR_BIN_SYMLINK_NAME@@"' ]; then
```

Comment:
There's no reason to prevent the stable package from providing an alternative for `/usr/bin/brave-browser` since it now uses a unique name for its binary (`/usr/bin/brave-browser-stable`).